packages/app/src/runner/event-manager.ts

```diff
@@ -3,19 +3,28 @@ import { EventEmitter } from 'events'
 import type { BaseStore } from '@packages/runner-shared/src/store'
 import type { RunState } from '@packages/types/src/driver'
 import type MobX from 'mobx'
-import type { LocalBusEmitsMap, LocalBusEventMap } from './event-manager-types'
-import type { FileDetails } from '@packages/types'
+import type { LocalBusEmitsMap, LocalBusEventMap, DriverToLocalBus, SocketToDriverMap } from './event-manager-types'
+import type { AutomationElementId, FileDetails } from '@packages/types'
 
 import { automation } from '@packages/runner-shared/src/automation'
 import { logger } from '@packages/runner-shared/src/logger'
 import type { Socket } from '@packages/socket/lib/browser'
+import { AutomationStatus, useRunnerUiStore } from '../store'
 
 // type is default export of '@packages/driver'
 // cannot import because it's not type safe and tsc throw many type errors.
 type $Cypress = any
 
 const noop = () => {}
 
+export const automationElementId: AutomationElementId = '__cypress-string' as const
+
+interface AddGlobalListenerOptions {
+    element: AutomationElementId
+    randomString: string
+    setAutomationStatus: (status: AutomationStatus) => void
```

Comment:
Do we need this prop, we are using it from `runnerUiStore` in this file and not from options

Reply:
removed